src/utils/tests/all-projects-support.test.ts

```diff
@@ -2,8 +2,12 @@
  * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
  * See the LICENSE file in the repository root folder for details.
  */
-
-import { calculateEndDate, projectDurationBuilder } from '../src/backend-supports/projects-get-all';
+import { WbsElementStatus } from '../lib/types/project-types';
```

Comment:
This import location doesn't look right to me. Lib is a generated folder, so you don't wanna be importing from it

Reply:
Changed to import { WBS_Element_Status } from '@prisma/client';